components/automate-ui/src/app/entities/cookbooks/cookbook-details.model.ts

```diff
@@ -0,0 +1,28 @@
+export interface CookbookDetails {
+    name: string;
+    cookbook_name: string;
```

Comment:
Per my earlier question, you indicated "name" is really the name and version combined. So could we change "name" to "name_and_version" to help people understand that?

Reply:
Yes so as per your suggestion mapped name which we got from API response with name_and_version.